kymatio/scattering1d/scattering1d.py

```diff
@@ -379,6 +385,15 @@ def forward(self, x):
                        oversampling=self.oversampling,
                        vectorize=self.vectorize,
                        size_scattering=size_scattering)
+
+        if self.vectorize:
+            scattering_shape = S.shape[-2:]
+            S = S.view(batch_shape + scattering_shape)
```

Comment:
reshape creates a view if the array is contiguous and copies if it is not.

`.view()` plus assigning shape in numpy will error if there was non-contiguity and a copy is necessary. In pytorch I wouldn't even be too sure they have already dealt with all the edge cases